sites/dotnet-guide/contents/tutorials/resharper-to-rider/plugins-for-developers/index.md

```diff
@@ -0,0 +1,32 @@
+---
+type: TutorialStep
+date: 2021-01-29
+title: Plug-ins
+technologies: [.net,csharp]
+products: [rider,resharper]
+topics: [ide]
+author: khalidabuhakmeh
+subtitle: Learn the basics of the Rider plug-in marketplace from a Visual Studio + ReSharper user's perspective.
+thumbnail: ./thumbnail.png
+---
+
+The Visual Studio extension ecosystem is a rich one, and it is where productivity tools like ReSharper have an opportunity to enhance the development experience of .NET developers everywhere. If you use any extension other than ReSharper, you're likely familiar with the **Manage Extensions** dialog.
+
+![Visual Studio Extensions dialog](./1-visual-studio-extensions.png)
+
+ReSharper users will likely also have seen the **ReSharper Extensions** dialog, accessible under `Extensions > ReSharper > Extension Manager...`.
+
+![Visual Studio ReSharper Extensions](./2-visual-studio-resharper-extensions.png)
+
+Let's talk about IDE plug-ins, as they are closest in concept to Visual Studio extensions. Rider's plug-ins are accessible via the **Plugins** section in **Settings**. Developers can find many of the ReSharper extensions in the JetBrains Marketplace. Since ReSharper is the engine that powers many of Rider's editor features, most of these plug-ins "just work".
+
+![JetBrains Rider Plug-in Marketplace dialog](./3-jetbrains-rider-plug-in-dialog.png)
+
+A bonus for Rider users is we aren't limited to plug-ins that primarily target .NET development. Since Rider is part of the IntelliJ family of products, developers can benefit from plug-ins designed for IntelliJ IDEA, PyCharm, GoLand, RubyMine, WebStorm, and CLion. JetBrains ships many of the features in Rider as plug-ins deriving from other products.
```

Comment:
```suggestion
A bonus for Rider users is we aren't limited to plugins that primarily target .NET development. Since Rider is part of the IntelliJ family of products, developers can benefit from plugins designed for IntelliJ IDEA, PyCharm, GoLand, RubyMine, WebStorm, and CLion. JetBrains ships many of the features in Rider as plugins deriving from other products.
```